docs/executor-custom.md

```diff
@@ -101,11 +162,209 @@ echo "http://google.pl" | kubectl testkube tests create --name example-google-te
 kubectl testkube tests run example-google-test
 ```
 
-That's all for the most basic executor example, you can look our internal projects for more examples
-and details how it's implemented:
+That's all for the most basic executor example, you can look our internal projects for more examples and details how it's implemented:
+
+- [Postman runner implementation](https://github.com/kubeshop/testkube-executor-postman/blob/main/pkg/runner/newman/newman.go)
+- [Cypress runner implementation](https://github.com/kubeshop/testkube-executor-cypress/blob/main/pkg/runner/cypress.go)
+- [Curl runner implementation](https://github.com/kubeshop/testkube-executor-curl/blob/main/pkg/runner/runner.go)
+
+
+# Creating executor in other programming language (than `go`)
+
+([You can find full commented code example here](https://github.com/kubeshop/testkube-executor-example-nodejs/blob/main/app.js))
+
+For go-based executors we've prepared a lot of handy functions (like printing valid outputs or wrappers around calling external processes)
```

Comment:
For go-based executors, we have prepared many handy functions, such as printing valid outputs or wrappers around calling external processes.